### ISSUE TITLE
having a "insert excalidraw diagram here" command in right click context menu of a note

### ISSUE THREAD

**Issue by aliamini87** [zsviczian/obsidian-excalidraw-plugin#587]
It will be very useful if I can right click in every where in a note and choose "embed an Excalidraw diagram" from context menu.

**Comment by zsviczian** [comment]
I'll check how to implement it. 
There are couple of command palette actions for this purpose already. Have you tried those?

![image](https://user-images.githubusercontent.com/14358394/163389740-d0201cb6-5dfe-4879-982d-b3f46d3ae900.png)

**Comment by aliamini87** [comment]
Yes. I'm aware of them. But right click context menu is a lot handier I think. 
Thank you very much.
